### PR TITLE
docs - misc pxf updates

### PR DESCRIPTION
**Pull request by lisakowen**
misc pxf doc updates (not related to any code changes):
- note setting of hive property required for filter pushdown
- add the following prerequisite:  time synchronization between segment hosts and external systems
- use example user config directory PXF_CONF=/usr/local/greenplum-pxf (was previously /etc/pxf/usercfg)
- add info about segment failure to troubleshooting section
- update some SDK docs (note that the pxf sdk section has been temporarily removed from the doc set, plan to rework some of this content)

### CODE REVIEW

File: gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb [hunk 1]
```diff
@@ -13,22 +13,29 @@ You must also identify any dependencies that your connector has on third-party c
...
+- Restart the PXF agent on each host
+
+Only the Greenplum Database administrative user can restart the PXF agent. 
+
+For example, if `PXF_CONF=/usr/local/greenplum-pxf`, these commands deploy the connector across the cluster:
 
 ``` shell
-gpadmin@gpmaster$ gpscp my-connector.jar -v -f seghostfile =:/etc/pxf/usercfg/lib
-gpadmin@gpmaster$ gpscp connector-dependency.jar -v -seghostfile /etc/pxf/usercfg/lib
+gpadmin@gpmaster$ scp my-connector.jar $PXF_CONF/lib/
```
Comment (frankgh) [inline gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb]: Do we need a cp command here instead?
```suggestion
gpadmin@gpmaster$ cp my-connector.jar $PXF_CONF/lib/
```

File: gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb [hunk 2]
```diff
@@ -13,22 +13,29 @@ You must also identify any dependencies that your connector has on third-party c
...
+gpadmin@gpmaster$ scp connector-dependency.jar $PXF_CONF/lib/
```
Comment (frankgh) [inline gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb]: same here?

File: gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb [hunk 3]
```diff
@@ -219,3 +219,7 @@ You can use the `PXF_JVM_OPTS` property to set other Java options as well.
 
 As described in previous sections, you must synchronize the updated PXF configuration to each Greenplum Database segment host and restart the PXF server on each host.
 
+## <a id="pxf-segfail"></a>Greenplum Database Segment Failure
```
Comment (frankgh) [inline gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb]: Maybe add a note that to recover from this failure a `pxf cluster stop` `pxf cluster start` should work. Or alternatively, we can mention that the pxf service should be recovered on the failing node, by manually killing the process and doing a `pxf start`

File: gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb (hunk 2 above)
Comment (lisakowen) [reply gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb]: yep, thanks for catching those!

File: gpdb-doc/markdown/pxf/init_pxf.html.md.erb [hunk 4]
```diff
@@ -32,9 +32,9 @@ PXF provides two management commands that you can use for initialization:
...
 
-PXF internal configuration files are located in `$GPHOME/pxf/conf`. You identify the PXF user configuration directory at initialization time via an environment variable named `$PXF_CONF`. If you do not set `$PXF_CONF` prior to initializing PXF, PXF may prompt you to accept or decline the default user configuration directory, `$HOME/pxf`, during the initialization process.
+PXF internal configuration files are located in your Greenplum Database installation in the `$GPHOME/pxf/conf` directory. You identify the PXF user configuration directory at PXF initialization time via an environment variable named `$PXF_CONF`. If you do not set `$PXF_CONF` prior to initializing PXF, PXF may prompt you to accept or decline the default user configuration directory, `$HOME/pxf`, during the initialization process.
 
-**Note**: The `gpadmin` user must have permission to either create, or write to, the specified `$PXF_CONF` directory.
+**Note**: Choose a `$PXF_CONF` directory location that is both outside of your Greenplum Database installation, and one that you back up.
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/init_pxf.html.md.erb]: Would recommend editing to :  Choose ... that you can back up, and ensure that it resides outside of your Greenplum Database Installation directory.

File: gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb [hunk 5]
```diff
@@ -13,22 +13,29 @@ You must also identify any dependencies that your connector has on third-party c
...
 
 
 ## <a id="deploy_to_cluster"></a>Deploying to the Greenplum Database Cluster
 
-When a Greenplum Database administrator deploys a new connector, they copy the connector JAR file and the (JAR) files of any dependencies to each Greenplum Database segment host, and then must restart the PXF agent on each host. Only the Greenplum Database administrative user can restart the PXF agent. 
+When a Greenplum Database administrator deploys a new connector, they:
 
-For example, if `seghostfile` contains a list, one-host-per-line, of the segment hosts in your Greenplum Database cluster and `PXF_CONF=/etc/pxf/usercfg`, these commands deploy the connector across the cluster:
+- Copy the connector JAR file and the (JAR) files of any dependencies to the Greenplum Database master host
+- Synchronize PXF configuration to each Greenplum Database segment host
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb]: PXF configuration -> the PXF configuration

File: gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb [hunk 6]
```diff
@@ -105,16 +112,16 @@ Perform the following procedure to deploy the *Demo* connector and to verify tha
...
-2. Copy the *Demo* connector JAR file that you previously built to the Greenplum Database master host. For example, to copy the JAR file to the `/tmp` directory, replace `PXFDEV_BASE` with the absolute path to your PXF development work area:
+2. Copy the *Demo* connector JAR file that you previously built to the `$PXF_CONF/lib` directory on the Greenplum Database master host. Replace `PXFDEV_BASE` with the absolute path to your PXF development work area:
 
     ``` shell
-    gpadmin@gpmaster$ scp user@devsystem:/PXFDEV_BASE/demo_example/build/libs/my-demo-connector.jar /tmp/
+    gpadmin@gpmaster$ scp user@devsystem:/PXFDEV_BASE/demo_example/build/libs/my-demo-connector.jar $PXF_CONF/lib/my-demo-connector.jar
     ```
 
-8. Copy the *Demo* connector JAR file to the PXF user runtime library directory on all Greenplum Database segment hosts. For example:
+8. Copy the JAR file to all Greenplum Database segment hosts by synchronizing PXF configuration. For example:
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/sdk/deploy_conn.html.md.erb]: PXF configuration -> the PXF configuration

**Pull request merged**